libs/ngx-mime/src/lib/help-dialog/help-dialog.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { MediaObserver } from "@angular/flex-layout";
+import { DebugElement } from "@angular/core";
+import { By } from "@angular/platform-browser";
+import { MatDialogRef } from "@angular/material/dialog";
+import { SharedModule } from "../shared/shared.module";
+import { HelpDialogComponent } from "./help-dialog.component";
+import { MimeViewerIntl } from "../core/intl/viewer-intl";
+import { MimeResizeService } from "../core/mime-resize-service/mime-resize.service";
+import { MimeDomHelper } from "../core/mime-dom-helper";
+import { FullscreenService } from "../core/fullscreen-service/fullscreen.service";
+import { MatDialogRefStub } from "../test/mat-dialog-ref-stub";
+import { ViewerService } from "../core/viewer-service/viewer.service";
+import { ViewerServiceStub } from "../test/viewer-service-stub";
+
+describe('HelpDialogComponent', () => {
+  let component: HelpDialogComponent;
+  let fixture: ComponentFixture<HelpDialogComponent>;
+  let mediaObserver: any;
+  let dialogRef: any;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [SharedModule],
+      declarations: [HelpDialogComponent],
```

Comment:
Since HelpDialog has it's own module, you don't need declarations here, and you can import `HelpDialogModule` instead of `SharedModule`.